components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -244,7 +249,12 @@ class MozillaSocorroService(
     }
 
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    internal fun readExtrasFromFile(file: File): HashMap<String, String> {
+    internal fun gsonUnescape(string: String): String {
+        return string.replace("""\\\\""", "\\").replace("""\n""", "\n").replace("""\t""", "\t")
```

Comment:
Strings return from GSON can only be un-escaped correctly using the triple-quoted raw string.  Not exactly sure why.